src/app/components/mnoe-users-list/mno-users-list.coffee

```diff
@@ -93,7 +99,6 @@
 
     displayCurrentState = () ->
       setAllUsersList()
-      fetchUsers(scope.users.nbItems, 0, scope.users.sortAttr)
```

Comment:
More of a general question here which would apply to both the users list and the organizations list: is `scope.pipe` always triggered during initialization no matter what? Just curious for the sake of the organizations/users being fetched in the first place.